src/pybind/mgr/influx/module.py

```diff
@@ -21,32 +21,45 @@
 class Module(MgrModule):
     MODULE_OPTIONS = [
         Option(name='hostname',
-               default=None),
+               default=None,
+               desc='Influx server hostname'),
```

Comment:
nit: some of these description use Influx, and other InfluxDB - we should stick to one

Reply:
let's use InfluxDB then.